resources/assets/enderio/lang/ru_RU.lang

```diff
@@ -761,26 +762,26 @@ tile.blockPainter.tooltip.detailed.line6=и большинства машин
 //Items
 item.itemYetaWrench.name=Гаечный ключ Yeta
 item.itemYetaWrench.tooltip.detailed.line1=Скрывает фасады труб, когда в руке
-item.itemYetaWrench.tooltip.detailed.line2=Shift-ПКМ, чтобы моментально демонтировать
+item.itemYetaWrench.tooltip.detailed.line2=SHIFT+ПКМ, чтобы моментально демонтировать
 item.itemYetaWrench.tooltip.detailed.line3=механизм или трубы
-item.itemYetaWrench.tooltip.detailed.line4=Shift-ЛКМ по трубе, чтобы
+item.itemYetaWrench.tooltip.detailed.line4=SHIFT+ЛКМ по трубе, чтобы
 item.itemYetaWrench.tooltip.detailed.line5=открыть интерфейс настроек
-item.itemYetaWrench.tooltip.detailed.line6=Используйте Shift-колёсико мыши или '%s', чтобы
+item.itemYetaWrench.tooltip.detailed.line6=Используйте SHIFT+колёсико мыши или '%s', чтобы
 item.itemYetaWrench.tooltip.detailed.line7=отображать только один вид труб
 
 item.itemMagnet.name=Электромагнит
 item.itemMagnet.tooltip.detailed.line1=Тянет ближайшие к Вам предметы
-item.itemMagnet.tooltip.detailed.line2=Shift-ПКМ, чтобы включить/выключить
-item.itemMagnet.tooltip.detailed.line3=Должен быть запитан, активен и находиться на
+item.itemMagnet.tooltip.detailed.line2=SHIFT+ПКМ, чтобы включить/выключить
+item.itemMagnet.tooltip.detailed.line3=Должен быть запитан энергией, активен и находиться на
 item.itemMagnet.tooltip.detailed.line4=панели быстрого доступа (или в слоте Baubles) для функционирования
```

Comment:
You are supposed to translate the _meaning_ of the _original_ text to the adequate native wording.
Translating separate words, or editing someone else's translation without checking the original source often leads to the misinterpretation and ends in a wrong translation.